### DISCUSSION TITLE
索引时所有照片都会提示这个错误而无法成功

### DISCUSSION THREAD

**Discussion by jxjxjhjh** [photoprism/photoprism#3733] [Q&A]
![Screenshot_2023-09-17-10-59-21-043_mark via](https://github.com/photoprism/photoprism/assets/22759318/888ba69e-4f35-4ca7-a067-0695a20b5a28)

**Comment by graciousgrey** [comment]
This looks like a problem with your database. 
Is this a new installation. Did you use [this guide](https://docs.photoprism.app/getting-started/docker-compose/) and our docker-compose.yml example for the installation?
